src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -851,6 +851,47 @@ static void _applyTextFill(SvgStyleProperty* style, Text* text, const Box& vBox)
 }
 
 
+static char* _processText(const char* text, SvgXmlSpace space)
```

Comment:
Too much ad-hoc style addition, Parsing logics everywhere. See `tvgSvgLoader`.  We need to consider a well-organized and structured implementation. Consolidated parser in one place might be better to go.